### DISCUSSION TITLE
Org/repository name don't appear soon enough

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44018] [Mobile] [answered]
Visit Microsoft/Power toys

And scroll
![Screenshot_20230111-165705.png](https://user-images.githubusercontent.com/2119212/211926781-4ffd7617-1c8c-4acd-8463-a3a9523cc872.png)
![Screenshot_20230111-165719.png](https://user-images.githubusercontent.com/2119212/211926791-b2f48128-deb5-4715-8d94-8e2c99f90c98.png)
![Screenshot_20230111-165729.png](https://user-images.githubusercontent.com/2119212/211926801-b8b2e104-b7ce-4139-b3b4-7390f29a1e20.png)
![Screenshot_20230111-165748.png](https://user-images.githubusercontent.com/2119212/211926810-fe11c9d1-8a75-4b68-81a0-7374149941aa.png)

Eventually the name will appear above the fold. It shouldn't require this much scrolling.

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref I passed this on to the team to be fixed. Thanks!
